core/themes/seven/css/style.css

```diff
@@ -789,16 +789,16 @@ input.form-search,
 input.form-file,
```

Comment:
looks like we're targeting more than just the select here, was this intentional?